pkg/checkmarx/checkmarx.go

```diff
@@ -564,11 +553,11 @@ func (sys *SystemInstance) GetScans(projectID int) (bool, []ScanStatus) {
 	data, err := sendRequest(sys, http.MethodGet, fmt.Sprintf("/sast/scans?%v", body.Encode()), nil, header)
 	if err != nil {
 		sys.logger.Errorf("Failed to fetch scans of project %v: %s", projectID, err)
```

Comment:
In some cases this `logger` output is removed, in others like this not. Is this on purpose?